src/odemis/gui/comp/radio.py

```diff
@@ -73,6 +73,14 @@ def SetValue(self, value):
         logging.debug("Set radio button control to %s", value)
         for btn in self.buttons:
             btn.SetToggle(btn.value == value)
+            self.SetColor(btn, value)
+
+    def SetColor(self, btn, value):
```

Comment:
The second argument should be just `active`. Then, inside the function, it can just check for `if active`.
Please, also fix the doc string.

`_reset_buttons()` should also call this function.

IMHO, probably the function could actually be changed in meaning a be called `SetActive(btn, active)`, and it would do both `SetToggle()` and `SetForegroundColour()`. Then, the callers could be simplified.